### PR TITLE
C# Guess The Word Workshop conversion to .Net Fiddle 

### PR DESCRIPTION
**Pull request by veronicagg**
Updating media and content files to refer to .Net Fiddle instead of Replit

### CODE REVIEW

File: content/english/csharp-guess-the-word/_index.md [hunk 1]
```diff
@@ -16,27 +16,27 @@ The goal of today's activity is to learn C# skills and incorporate your new-foun
 
 The goal of today's activity is to learn C# skills and incorporate your new-found knowledge to create a Guess The Word game! Let's get the game set up.
 
-Today you will be using Replit. Before starting, you will need to sign up for a Replit account, if you do not yet have one. 
+Today you will be using .NET Fiddle. Before starting, you will need to sign up for a .NET Fiddle account, if you do not yet have one. 
 
-Click the button below to sign up for a Replit account - you will need an email account in order to sign up. If you already have a Replit account, you can login to your account using the below button as well.
```
Comment (ozhang22) [inline content/english/csharp-guess-the-word/_index.md:21]: Let's mark the sign up process as optional by surrounding with a {{ % notice tip % }} and {{ % /notice % }}

File: content/english/csharp-guess-the-word/boolean-operators.md [hunk 2]
```diff
@@ -35,6 +35,6 @@ Try guessing the answers to the following expressions. Use `Console.WriteLine` t
 - `(9 < 10) && (12 => 11)`
 - `(15 - 2 == 11) || (4 % 3 != 2)`
 
-<a class="my-2 mx-4 btn btn-info" href="https://replit.com/@nuevofoundation/NF-CSharp-blank" target="_blank">Launch Replit</a>
+<a class="my-2 mx-4 btn btn-info" href="https://dotnetfiddle.net/ULv0JH" target="_blank">Launch Fiddle</a>
```
Comment (ozhang22) [inline content/english/csharp-guess-the-word/boolean-operators.md:38]: For all of these fiddles that are not found in the starting up page - these were actually previously embedded directly into the workshop before replit stopped supporting embeds - is this something that we can reenable?

File: content/english/csharp-guess-the-word/starting-up.md
Comment (ozhang22) [inline content/english/csharp-guess-the-word/starting-up.md:1]: This is weird... this file is duplicative of the _index file... Let's delete this.

File: content/english/csharp-guess-the-word/activity-1.md [hunk 3]
```diff
@@ -7,8 +7,8 @@ weight: 4
 
 ## Instructions
 
-Go to the code containing the [GuessTheWord](https://aka.ms/nuevoguesstheword) game. Press **run**. Notice that `Press 'run' on the top of the screen to play again.` is the only thing that is printed to the console. Let's add a welcome message so that the player knows they playing the Guess the Word game.
+Go to the code containing the [GuessTheWord](https://dotnetfiddle.net/lMl7j4) game. Press **run**. Notice that `Press 'run' on the top of the screen to play again.` is the only thing that is printed to the console. Let's add a welcome message so that the player knows they playing the Guess the Word game.
```
Comment (ozhang22) [inline content/english/csharp-guess-the-word/activity-1.md:10]: Could you change the link to be of one that opens in a new page?

```
<a href="https://dotnetfiddle.net/lMl7j4" target="_blank">Guess The Word </a>
```

File: content/english/csharp-guess-the-word/_index.md [hunk 4]
```diff
@@ -16,27 +16,36 @@ The goal of today's activity is to learn C# skills and incorporate your new-foun
 
 The goal of today's activity is to learn C# skills and incorporate your new-found knowledge to create a Guess The Word game! Let's get the game set up.
 
-Today you will be using Replit. Before starting, you will need to sign up for a Replit account, if you do not yet have one. 
+Today you will be using .NET Fiddle.
```
Comment (ozhang22) [inline content/english/csharp-guess-the-word/_index.md:19]: Maybe add a small explanation of what .NET Fiddle is?

**Review by ozhang22 — APPROVED** [review]

File: content/english/csharp-guess-the-word/activity-1.md (hunk 3 above)
Comment (veronicagg) [reply content/english/csharp-guess-the-word/activity-1.md:10]: clicking on the link does open it in a new window when I try it, so I think we could leave it as is.

**Pull request merged**